docs/routing.md

```diff
@@ -262,3 +263,42 @@ app = Router(routes=[
     ])
 ])
 ```
+
+## WebSocket Routing
+
+When working with WebSocket endpoints, you should use `WebSocketRoute`
+instead of the usual `Route`.
+
+Path parameters, and reverse URL lookups for `WebSocketRoute` work the the same
+as HTTP `Route`, which can be found in the [Route](#path-parameters) documentation.
```

Comment:
Looks weird talking about it like if it was in another location, but in reality, it's actually above :thinking:

Can we rephrase this last part (after the last comma)? Something like "like mentioned in the section [Path parameters](#path-parameters) above", or something like that?

If you disagree, feel free to close this thread :+1:

Reply:
I agree with making it obvious on the same page.
Another point, since we're linking multiple things (path params, url lookups, ..), what do you think instead we link it to the `HTTP Routing` section? It's not really accurate to point to path parameters.

Reply:
Sure :+1: